beacon-chain/rpc/prysm/v1alpha1/validator/proposer_epbs_test.go

```diff
@@ -0,0 +1,99 @@
+package validator
+
+import (
+	"context"
+	"testing"
+
+	chainMock "github.com/prysmaticlabs/prysm/v5/beacon-chain/blockchain/testing"
+	"github.com/prysmaticlabs/prysm/v5/beacon-chain/cache"
+	powtesting "github.com/prysmaticlabs/prysm/v5/beacon-chain/execution/testing"
+	doublylinkedtree "github.com/prysmaticlabs/prysm/v5/beacon-chain/forkchoice/doubly-linked-tree"
+	mockSync "github.com/prysmaticlabs/prysm/v5/beacon-chain/sync/initial-sync/testing"
+	"github.com/prysmaticlabs/prysm/v5/config/params"
+	"github.com/prysmaticlabs/prysm/v5/consensus-types/blocks"
+	"github.com/prysmaticlabs/prysm/v5/consensus-types/primitives"
+	"github.com/prysmaticlabs/prysm/v5/encoding/ssz"
+	v1 "github.com/prysmaticlabs/prysm/v5/proto/engine/v1"
+	eth "github.com/prysmaticlabs/prysm/v5/proto/prysm/v1alpha1"
+	"github.com/prysmaticlabs/prysm/v5/testing/require"
+	"github.com/prysmaticlabs/prysm/v5/testing/util"
+)
+
+func TestServer_GetLocalHeader(t *testing.T) {
+	t.Run("Node is syncing", func(t *testing.T) {
+		vs := &Server{
+			SyncChecker: &mockSync.Sync{IsSyncing: true},
+		}
+		_, err := vs.GetLocalHeader(context.Background(), &eth.HeaderRequest{
+			Slot:          0,
+			ProposerIndex: 0,
+		})
+		require.ErrorContains(t, "Syncing to latest head, not ready to respond", err)
+	})
+	t.Run("ePBS fork has not occurred", func(t *testing.T) {
+		vs := &Server{
+			SyncChecker: &mockSync.Sync{IsSyncing: false},
+			TimeFetcher: &chainMock.ChainService{},
+		}
+		_, err := vs.GetLocalHeader(context.Background(), &eth.HeaderRequest{
+			Slot:          0,
+			ProposerIndex: 0,
+		})
+		require.ErrorContains(t, "EPBS fork has not occurred yet", err)
+	})
+	t.Run("Happy case", func(t *testing.T) {
+		params.SetupTestConfigCleanup(t)
+		cfg := params.BeaconConfig().Copy()
```

Comment:
This uses minimal config, producing the following error: `Unexpected error: rpc error: code = Internal desc = Could not process slots up to 8: could not process slots: could not process epbs epoch: state randao length 65536 different than EpochsPerHistoricalVector 64`.

I came up with two solutions:
1) Replace L48 with `cfg := params.MainnetConfig().Copy()` and add `OptimisticModeFetcher:  &mock.ChainService{Optimistic: false},` to `vs := &Server{...}` (L70 or so).
2) Add `cfg.EpochsPerHistoricalVector = 65536` between L48 and L49.

Reply:
CI/CD passes, that's all it matters here. To run `beacon-chain/rpc/prysm/v1alpha1/validator`, you should use bazel instead of native golang.  `bazel test //beacon-chain/rpc/prysm/v1alpha1/validator:go_default_test --test_filter=TestServer_GetLocalHeader`